src/test/isolation2/sql/fts_segment_reset.sql

```diff
@@ -12,15 +12,15 @@
 -- to restart, and potentially makes FTS think it's in "recovery not
 -- in progress" stage and promote the mirror, we would need the FTS
 -- to make that decision a bit less frequently.
-!\retcode gpconfig -c gp_fts_probe_retries -v 15 --masteronly;
+!\retcode gpconfig -c gp_fts_probe_retries -v 15;
```

Comment:
Why is this change needed? FTS runs on the master only.

Reply:
Because the crash will be handled by segment, and we need to know `gp_fts_probe_retries` on the segment to set properly `delay_bg_writer_termination_sec`. So we need to propagate this  guc to the segments as well.